src/surveys/components/ChoiceResponse.tsx

```diff
@@ -0,0 +1,85 @@
+import { VNode, h } from 'preact';
+import { useState } from 'preact/hooks';
+import { Button, Header } from '../../components';
+import { FormSettings, ID, Question, Theme } from '../../types';
+
+interface SingleResponseProps {
+  question: Question;
+  onAnswered: (questionId: ID, answerId: ID | ID[]) => void;
+  submitText?: string;
+  theme?: Theme;
+}
+
+export default function SelectResponse({
+  question,
+  onAnswered,
+  submitText,
+  theme,
+}: SingleResponseProps): VNode {
+  const [selectedOption, setSelectedOption] = useState<ID | ID[]>(
+    (question.settings as FormSettings).multiselect ? [] : ''
```

Comment:
@afolasope Please use the `question.type` to determine the response type instead of creating a new one.